src/shared/ui/surfaces/Card/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from "next/image";
+import { IoMdPlay } from "react-icons/io";
+import { twJoin } from "tailwind-merge";
+
+import { IconButton } from "../../inputs";
+
+import { Paper } from "..";
+
+export interface CardProps {
+  /**
+   * Whether or not to round the top corners of the image.
+   */
+  roundedPicture?: boolean;
+}
+
+/**
+ * Renders a Card component with an image, title, and description.
+ */
+export function Card({ roundedPicture }: CardProps) {
+  return (
+    <div className="group p-spacing-looser">
+      <Paper className="p-spacing-base" variant="tinted">
+        <div className="relative">
+          <Image
+            src="https://picsum.photos/300/300"
+            width={300}
+            height={300}
+            alt="card picture"
+            className={twJoin(
+              `rounded-border-radius-larger`,
+              roundedPicture && `rounded-full`,
+            )}
+          />
+          <IconButton
+            className="theme-accent absolute bottom-spacing-tighter-2 right-spacing-tighter-2 translate-y-spacing-tighter-2 bg-background-base p-spacing-tighter opacity-0 transition-all duration-300 hover:scale-105 hover:bg-background-highlight group-hover:translate-y-0 group-hover:opacity-100"
+            icon={
+              <IoMdPlay className="size-4 text-text-base" aria-hidden="true" />
+            }
+          />
+        </div>
+        <div className="mt-spacing-base">
+          <a href="" className="font-bold">
+            Todays Top Hits
+          </a>
+          <span className="theme-muted-accent line-clamp-2 text-font-size-smaller font-semibold text-essential-subdued">
```

Comment:
Here, I think we can remove `theme-muted-accent` and apply text-text-subdued` instead.

`essential` classes seem to be reserved for other ui elements, but not texts. (it is applied on the switch component for example.

Reply:
Yes well observed, modified !